packages/client/src/App.js

```diff
@@ -7,12 +7,12 @@ import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
 function App() {
   return (
     <div className="app">
-      <Router>
-        <Routes>
-          <Route path="/" element={<LandingPage />} />
-          <Route path="*" element={<PageNotFound />} />
-        </Routes>
-      </Router>
+        <Router>
+          <Routes>
+            <Route path="/" element={<LandingPage />} />
+            <Route path="*" element={<PageNotFound />} />
+          </Routes>
+        </Router>
```

Comment:
I think you need to run prettier on this file. You added some spaces that are breaking the prettier check